test/tls_test.go

```diff
@@ -28,26 +28,57 @@ func TestEnablePachTLSNoName(t *testing.T) {
 }
 
 func TestEnablePachTLSExistingSecret(t *testing.T) {
-	helmChartPath := "../pachyderm"
+	expectedSecretName := "blah"
 	options := &helm.Options{
 		SetValues: map[string]string{
 			"deployTarget":         "LOCAL",
 			"pachd.tls.enabled":    "true",
 			"pachd.tls.secretName": "blah",
 		},
 	}
+	templates := []string{"templates/pachd/deployment.yaml"}
+	output := helm.RenderTemplate(t, options, "../pachyderm", "blah", templates)
```

Comment:
This hardcodes a constant `"../pachyderm"` for the path to the Helm chart, while tests below refer to locally-scoped constants. Probably worth pulling out into a _package_ constant, I think. Unexported, of course!